src/mantine-demos/src/demos/form/Form.demo.valibot.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-console */
+import { MantineDemo } from '@mantine/ds';
+import { SchemaBase } from './_schema-base';
+
+const code = `
+import { Box, Button, Group, NumberInput, TextInput } from '@mantine/core';
+import { useForm, valibotResolver } from '@mantine/form';
+import { minLength, object, string } from 'valibot';
+
+const schema = object({
+  username: string('Username is required', [
+    minLength(8, 'Username must have at least 8 characters'),
+  ]),
+  password: string('Password is required', [
+    minLength(8, 'Password must have at least 8 characters'),
+  ]),
+});
+
+function Demo() {
+  const form = useForm({
```

Comment:
In typescript, we might need the generic `<FormData>` type. To type it in valibot:

```ts
import { type Input } from 'valibot';

type FormData = Input<typeof schema>;
```

Reply:
Ah, I just revisited the [docs](https://mantine.dev/form/validation/#zod), it doesn't specify the generic typing usage though it's a `.tsx` file. I also forgot where I saw that usage. I think we can drop it then.

Reply:
Sure, removed it